nixos/modules/rename.nix

```diff
@@ -240,6 +240,11 @@ with lib;
 
     # binfmt
     (mkRenamedOptionModule [ "boot" "binfmtMiscRegistrations" ] [ "boot" "binfmt" "registrations" ])
+    
+    # ACME
+    (mkRemovedOptionModule [ "security" "acme" "directory"] "ACME Directory is now hardcoded to /var/lib/acme and its permisisons are managed by systemd. See https://github.com/NixOS/nixpkgs/issues/53852 for more info.")
```

Comment:
```suggestion
    (mkRemovedOptionModule [ "security" "acme" "directory"] "ACME Directory is now hard-coded to /var/lib/acme and its permissions are managed by systemd. See https://github.com/NixOS/nixpkgs/issues/53852 for more info.")
```

Reply:
I was wondering whether this line even works in the first place.
I did not remove the `directory` option. I made it `readOnly` instead.

I honestly have no clue how the `rename.nix` module works. so perhaps this is totally wrong in the first place?